fboss/platform/configs/icetea/sensor_service.json

```
@@ -1802,7 +1822,7 @@
           "sysfsPath": "/run/devmap/sensors/SMB_ADC2_SENSOR/in1_input",
           "type": 1,
           "thresholds": {
-            "maxAlarmVal": 1.236,
+            "maxAlarmVal": 1.30,
```

1.30 failed json validation
```suggestion
            "maxAlarmVal": 1.3,
```